modules/tinymce/src/core/test/ts/browser/ForceBlocksTest.ts

```diff
@@ -19,66 +22,74 @@ UnitTest.asynctest('browser.tinymce.core.ForceBlocksTest', (success, failure) =>
     dom.fire(target, 'keyup', evt);
   };
 
-  suite.test('Wrap single root text node in P', (editor) => {
+  it('Wrap single root text node in P', () => {
+    const editor = hook.editor();
     editor.focus();
     editor.getBody().innerHTML = 'abcd';
     LegacyUnit.setSelection(editor, 'body', 2);
     pressArrowKey(editor);
-    LegacyUnit.equal(HtmlUtils.cleanHtml(editor.getBody().innerHTML), '<p>abcd</p>');
-    LegacyUnit.equal(editor.selection.getNode().nodeName, 'P');
+    assert.equal(HtmlUtils.cleanHtml(editor.getBody().innerHTML), '<p>abcd</p>');
+    assert.equal(editor.selection.getNode().nodeName, 'P');
   });
 
-  suite.test('Wrap single root text node in P with attrs', (editor) => {
+  it('Wrap single root text node in P with attrs', () => {
+    const editor = hook.editor();
     editor.settings.forced_root_block_attrs = { class: 'class1' };
     editor.getBody().innerHTML = 'abcd';
     LegacyUnit.setSelection(editor, 'body', 2);
     pressArrowKey(editor);
-    LegacyUnit.equal(editor.getContent(), '<p class="class1">abcd</p>');
-    LegacyUnit.equal(editor.selection.getNode().nodeName, 'P');
+    assert.equal(editor.getContent(), '<p class="class1">abcd</p>');
```

Comment:
I've been skimming through and seeing these where we could do TinyAssertions.assertContent. I assume these weren't changed because of a regex find/replace - if that's the case we can probably replace them with a new one so I won't leave any more comments on them (and I'm sure I missed plenty anyway)

Reply:
As discussed on our call, we'll follow that up later in another PR.